Update add-site.mdx (#16959)
cloudflare/cloudflare-docs: src/content/docs/fundamentals/setup/manage-domains/add-site.mdx

import { Render } from "~/components"

> Follow these steps to add your website or application to Cloudflare. This process allows you to [proxy web traffic](/fundamentals/concepts/how-cloudflare-works/#how-cloudflare-works-as-a-reverse-proxy) through Cloudflare, which speeds up and protects websites and services on your domain.

Author: ```suggestion
Follow these steps to add your website or application to Cloudflare. Once your domain is [active](/dns/zone-setups/reference/domain-status/#active), your web traffic will [proxy through Cloudflare](/fundamentals/concepts/how-cloudflare-works/#how-cloudflare-works-as-a-reverse-proxy), which speeds up and protects websites and services on your domain.
```

> Follow these steps to add your website or application to Cloudflare. This process allows you to [proxy web traffic](https://developers.cloudflare.com/fundamentals/concepts/how-cloudflare-works/#how-cloudflare-works-as-a-reverse-proxy) through Cloudflare, which speeds up and protects websites and services on your domain.

Author: ```suggestion
Follow these steps to add your website or application to Cloudflare. This process allows you to [proxy web traffic](/fundamentals/concepts/how-cloudflare-works/#how-cloudflare-works-as-a-reverse-proxy) through Cloudflare, which speeds up and protects websites and services on your domain.
```
